man/Dockerfile.aarch64

```diff
@@ -1,6 +1,25 @@
-FROM    aarch64/ubuntu:xenial
+FROM    aarch64/ubuntu:wily
```

Comment:
align on Xenial is preferable since it's the latest LTS release and most likely deployed on servers along docker.